src/validation/validation.cpp

```diff
@@ -481,7 +481,7 @@ bool LoadBlockIndexDB()
     // Check whether we need to continue reindexing
     bool fReindexing = false;
     pblocktree->ReadReindexing(fReindexing);
-    fReindex |= fReindexing;
+    fReindex = fReindexing;
```

Comment:
The |= allows the reindexing signal to come from 2 places: either the pblocktree OR the fReindex variable.  I'd guess this is here because fReindex is set before this line from the command line or bitcoin.conf, so we need to preserve the |= semantics.

Reply:
yes true,  fixed...